src/app.js

```diff
@@ -276,11 +276,16 @@ export function app(app) {
           removeElement(element, reusableChild[0], reusableNode)
         }
       }
-    } else if (node !== oldNode) {
+    } else if (
+      element != null &&
+      node !== oldNode &&
+      node !== element.nodeValue
+    ) {
       var i = element
       parent.replaceChild((element = createElement(node)), i)
```

Comment:
beware of all perf-related conclusions reached > 3 months ago ;)

https://jsperf.com/innerhtml-vs-innertext/7.atom.atom.atom

the difference is *massive* in Edge and Firefox. Less so in Chrome, but still..